src/stubs.ts

```diff
@@ -288,14 +288,36 @@ export const createFigma = (config: TConfig): PluginAPI => {
     }
   }
 
+  class GeometryMixinStub implements GeometryMixin {
+    private _fills: ReadonlyArray<Paint> | PluginAPI["mixed"];
+    get fills() {
+      return this._fills || [];
+    }
+    set fills(theFills) {
+      this._fills = theFills;
+    }
```

Comment:
I tried to use  `fills = []` but it didn't work out, unfortunately. Seems to be related to the implementation of `applyMixins`